src/chain_spec.rs

```diff
@@ -504,21 +531,47 @@ fn alcyone_testnet_genesis(
             code: alcyone::WASM_BINARY.to_vec(),
             changes_trie_config: Default::default(),
         }),
-        asset: Some(AlcyoneConfig::AssetConfig {
-            ticker_registration_config: TickerRegistrationConfig {
-                max_ticker_length: 12,
-                registration_length: Some(5_184_000_000),
-            },
-            classic_migration_tconfig: TickerRegistrationConfig {
-                max_ticker_length: 12,
-                // TODO(centril): use values per product team wishes.
-                registration_length: Some(5_184_000_000),
-            },
-            // TODO(product_team): Assign to a real person.
-            classic_migration_contract_did: IdentityId::from(1),
-            // TODO(centril): fill with actual data from Ethereum.
-            classic_migration_tickers: vec![],
-        }),
+        asset: {
+            let currency_codes: Vec<Vec<u8>> = vec![
```

Comment:
If we encode the currency list in code, it would be preferable imo to encode it once in `chain_spec.rs` via some function, e.g., `fn currency_codes() -> Vec<Ticker> { ... }` and then just `reserved_country_currency_codes: currency_codes(),`.

Reply:
Alright I would work on that